Provide a method for supressing CPAN's version check.
Patch from Robert Bihlmeyer <robbe@orcus.priv.at> (Bug#98235)

diff -Naur --exclude=debian perl-5.8.7.orig/lib/CPAN/FirstTime.pm perl-5.8.7/lib/CPAN/FirstTime.pm
--- perl-5.8.7.orig/lib/CPAN/FirstTime.pm	2003-07-30 02:05:27.000000000 +1000
+++ perl-5.8.7/lib/CPAN/FirstTime.pm	2005-06-03 00:13:59.000000000 +1000
@@ -464,6 +464,7 @@
 
     # We don't ask that now, it will be noticed in time, won't it?
     $CPAN::Config->{'inhibit_startup_message'} = 0;
+    $CPAN::Config->{'cpan_version_check'} = 1;
     $CPAN::Config->{'getcwd'} = 'cwd';
 
     print "\n\n";
diff -Naur --exclude=debian perl-5.8.7.orig/lib/CPAN.pm perl-5.8.7/lib/CPAN.pm
--- perl-5.8.7.orig/lib/CPAN.pm	2003-09-22 15:42:59.000000000 +1000
+++ perl-5.8.7/lib/CPAN.pm	2005-06-03 00:13:59.000000000 +1000
@@ -3291,7 +3291,8 @@
 	      )
 	   ) {
             local($^W)= 0;
-            if ($version > $CPAN::VERSION){
+            if ($CPAN::Config->{cpan_version_check} &&
+	      $version > $CPAN::VERSION){
                 $CPAN::Frontend->myprint(qq{
   There's a new CPAN.pm version (v$version) available!
   [Current version is v$CPAN::VERSION]
@@ -6765,6 +6766,7 @@
   index_expire       after this many days refetch index files
   cache_metadata     use serializer to cache metadata
   cpan_home          local directory reserved for this package
+  cpan_version_check if true, warns you when the CPAN module is out of date.
   dontload_hash      anonymous hash: modules in the keys will not be
                      loaded by the CPAN::has_inst() routine
   gzip		     location of external program gzip
